src/stellar/stellarEngine.ts

```diff
@@ -385,19 +384,19 @@ export class StellarEngine extends CurrencyEngine<StellarPlugin> {
       }
     }
 
-    if (bns.eq(nativeAmount, '0')) {
+    if (eq(nativeAmount, '0')) {
       throw new NoAmountSpecifiedError()
     }
 
-    const exchangeAmount = bns.div(nativeAmount, denom.multiplier, 7)
+    const exchangeAmount = div(nativeAmount, denom.multiplier, 7)
 
     const account = new this.stellarApi.Account(
       this.walletLocalData.publicKey,
       this.otherData.accountSequence
     )
     let memoId: ?string
     if (
-      edgeSpendInfo.spendTargets[0].otherParams &&
+      edgeSpendInfo.spendTargets[0].otherParams != null &&
```

Comment:
Replace entire condition with optional chaining

Reply:
The goal of this PR is not to fix all the possible TS errors but to just not be "worse than Flow". A future commit auto adds the appropriate eslint exception comment which we can remove in future work and slowly clean up. Otherwise you'd be making 100s of these type of comments